src/index.js

```diff
@@ -20,36 +21,33 @@ const { REACT_APP_DHIS2_BASE_URL } = process.env
 
 init({
```

Comment:
Is there a reason that we don't do this in `componentDidMount`? That way we could already render a spinner. I assume you're just sticking to the existing approach here?

Reply:
I've just copied our existing approaches so far.
I also want to reduce the amount of work to a minimum because this one is time critical